### ISSUE TITLE
Need to update default color mode for V2

### ISSUE THREAD

**Issue by captain-nimo** [hackforla/311-data-archive#1123]
# Description

Currently, the default theme is dark, we need to change the default color theme/mode.

## Action Items

- [ ] Make light color as the default mode.
- [ ] Users should be able to change color mode to dark mode (if they want to).
